### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RSP531**
added the scripts for deploy and restart

### CODE REVIEW

File: client/App.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ class App extends React.Component {
         this.setState({option_text: event.target.value})
     }
 
+
```
Comment (devinkgeorge) [inline client/App.js]: Why did you add this space?

Comment (RSP531) [reply client/App.js]: because you suck

Comment (RSP531) [reply client/App.js]: I have no clue...
